lib/config/get.js

```diff
@@ -1,10 +1,24 @@
 const path = require('path')
-const { getInstalledPathSync } = require('get-installed-path')
+const { curry, flow, get } = require('lodash/fp')
+// const { getInstalledPathSync } = require('get-installed-path')
+
+// Return the appPath from an app object.
+const getAppPath = get('config.appPath')
+
+// Build path to the public dir of a module.
+const getModulePublic = curry((moduleName, app) =>
+  path.join(getAppPath(app), 'node_modules', moduleName, 'public')
+)
+// path.join(getInstalledPathSync(moduleName, { local: true }), 'public')
 
 // #521 Returns path to load plugin-config assets.
-const getPluginConfigPublic = () =>
-  path.join(getInstalledPathSync('@signalk/plugin-config'), 'public')
+const getPluginConfigPublic = getModulePublic('@signalk/plugin-config')
+// #524 Returns path to load admin-ui client assets.
+const getAdminPublic = getModulePublic('@signalk/admin-ui')
```

Comment:
I vote for removing `server-admin-ui.js` altogether and making the admin ui a plugin.

Reply:
My issue was not being able to test the `getAdminPublic` function because the the interface occupies the entire `module.exports`. Maybe I could have made a const for the function and added properties to that and then exported the result...